pkg/mount/mount.go

```diff
@@ -323,12 +322,17 @@ func (m *Mounter) deletePath(path string) bool {
 // Mount new mountpoint for specified device.
 func (m *Mounter) Mount(
 	minor int,
-	device, path, fs string,
+	devPath, path, fs string,
 	flags uintptr,
 	data string,
 	timeout int,
 	opts map[string]string,
 ) error {
+	device := devPath
```

Comment:
Could you add a comment here that device is initialized to devPath, but could be overwritten by opts[]?